src/openrct2/scenario/scenario.h

```diff
@@ -368,7 +381,7 @@ extern rct_s6_info gS6Info;
 extern char gScenarioName[64];
 extern char gScenarioDetails[256];
 extern char gScenarioCompletedBy[32];
-extern char gScenarioSavePath[MAX_PATH];
+extern char gScenarioSavePath[260];
```

Comment:
I don't like this change, Linux and other systems have larger path limits and we should start supporting it.